src/containers/createRoom.js

```diff
@@ -9,35 +9,91 @@ export class createRoom extends Component {
     username: "",
   };
 
-  handleChange = (event) => {
+  handleChange(event) {
     this.setState({ [event.target.name]: event.target.value });
   };
```

Comment:
The modified form is more idiomatic javascript class method syntax.

Reply:
no